src/graphql/pull_request.rs

```diff
@@ -3,36 +3,232 @@ use std::fmt;
+}
+
+#[derive(SimpleObject)]
+pub(crate) struct Label {
+    pub(crate) name: String,
+}
+
+#[derive(SimpleObject, Debug)]
+pub(crate) struct Comment {
+    pub(crate) body: String,
+    #[graphql(name = "createdAt")]
+    pub(crate) created_at: DateTimeUtc,
+    #[graphql(name = "updatedAt")]
+    pub(crate) updated_at: DateTimeUtc,
+    pub(crate) author: Option<User>,
+}
+
+#[derive(SimpleObject)]
+pub(crate) struct Review {
+    pub(crate) author: Option<User>,
+    pub(crate) state: PullRequestReviewState,
+    pub(crate) body: Option<String>,
+    pub(crate) url: String,
+    #[graphql(name = "createdAt")]
+    pub(crate) created_at: DateTimeUtc,
+    #[graphql(name = "publishedAt")]
+    pub(crate) published_at: DateTimeUtc,
+    #[graphql(name = "submittedAt")]
+    pub(crate) submitted_at: DateTimeUtc,
+    #[graphql(name = "isMinimized")]
+    pub(crate) is_minimized: bool,
+    pub(crate) comments: Vec<Comment>,
+}
+
+#[derive(SimpleObject)]
+pub(crate) struct CommitInfo {
+    pub(crate) additions: i32,
+    pub(crate) deletions: i32,
+    pub(crate) message: String,
+    #[graphql(name = "messageBody")]
+    pub(crate) message_body: Option<String>,
+    pub(crate) author: Option<User>,
+    #[graphql(name = "changedFilesIfAvailable")]
+    pub(crate) changed_files_if_available: Option<i32>,
+    #[graphql(name = "committedDate")]
+    pub(crate) committed_date: DateTimeUtc,
+    pub(crate) committer: Option<User>,
+}
 
 #[derive(SimpleObject)]
 pub(crate) struct PullRequest {
-    pub(crate) owner: String,
-    pub(crate) repo: String,
+    pub(crate) id: ID,
     pub(crate) number: i32,
     pub(crate) title: String,
-    pub(crate) assignees: Vec<String>,
-    pub(crate) reviewers: Vec<String>,
+    pub(crate) body: Option<String>,
+    pub(crate) state: PullRequestState,
+    #[graphql(name = "assignees")]
+    pub(crate) assignees: Vec<User>,
+    #[graphql(name = "createdAt")]
+    pub(crate) created_at: DateTimeUtc,
+    #[graphql(name = "updatedAt")]
+    pub(crate) updated_at: DateTimeUtc,
+    #[graphql(name = "closedAt")]
+    pub(crate) closed_at: Option<DateTimeUtc>,
+    #[graphql(name = "mergedAt")]
+    pub(crate) merged_at: Option<DateTimeUtc>,
+    pub(crate) author: Option<User>,
+    pub(crate) additions: i32,
+    pub(crate) deletions: i32,
+    pub(crate) url: String,
+    pub(crate) repository_owner: User,
+    pub(crate) repository_name: String,
+    pub(crate) labels: Vec<Label>,
+    #[graphql(name = "commentsCount")]
+    pub(crate) comments_count: i32,
+    pub(crate) comments: Vec<Comment>,
+    #[graphql(name = "reviewDecision")]
+    pub(crate) review_decision: Option<PullRequestReviewState>,
+    #[graphql(name = "reviewRequests")]
+    pub(crate) review_requests: Vec<User>,
+    #[graphql(name = "reviewsCount")]
+    pub(crate) reviews_count: i32,
+    pub(crate) reviews: Vec<Review>,
+    #[graphql(name = "commitsCount")]
+    pub(crate) commits_count: i32,
+    pub(crate) commits: Vec<CommitInfo>,
```

Comment:
Unlike the case of `Comment` and `CommentNode`, you've implemented this without defining separate structs. From the user's perspective, I think this could potentially lead to confusion, especially if the number of fields differs between `GitHubPullRequests` and `PullRequest`. 

What are your thoughts on this? Personally, I'd recommend clearly separating these into distinct structs, similarly to what we've done previously, to avoid ambiguity.